tests/regr_test.py

```diff
@@ -202,39 +203,103 @@ def run_testcases(
 
     mypy_command = [python_exe, "-m", "mypy"] + flags
     if verbosity is Verbosity.VERBOSE:
-        verbose_log(f"{mypy_command=}")
+        description = f"{package.name}/{version}/{platform}"
+        msg = f"{description}: {mypy_command=}\n"
         if "MYPYPATH" in env_vars:
-            verbose_log(f"{env_vars['MYPYPATH']=}")
+            msg += f"{description}: {env_vars['MYPYPATH']=}"
         else:
-            verbose_log("MYPYPATH not set")
+            msg += f"{description}: MYPYPATH not set"
+        msg += "\n"
+        verbose_log(msg)
     return subprocess.run(mypy_command, capture_output=True, text=True, env=env_vars)
 
 
-def test_testcase_directory(
-    package: PackageInfo, version: str, platform: str, *, verbosity: Verbosity, tempdir: Path
-) -> ReturnCode:
-    msg = f"Running mypy --platform {platform} --python-version {version} on the "
-    msg += "standard library test cases..." if package.is_stdlib else f"test cases for {package.name!r}..."
+@dataclass(frozen=True)
+class Result:
+    code: int
+    command_run: str
+    stderr: str
+    stdout: str
+    test_case_dir: Path
+    tempdir: Path
+
+    def print_description(self, *, verbosity: Verbosity) -> None:
```

Comment:
@AlexWaygood After this refactor, the `verbosity` argument of `Result.print_description` has been left unused. Was that intentional ?

Reply:
Nope!